src/gui/MainWindow.cpp

```diff
@@ -409,6 +411,10 @@ void MainWindow::initGui()
 	mSaveAsAction->setIcon(iconLoader->loadForTheme(QLatin1String("saveAs")));
 	connect(mSaveAsAction, &QAction::triggered, this, &MainWindow::saveAsClicked);
 
+	mSaveAllAction->setText(tr("Save All..."));
+	mSaveAllAction->setIcon(iconLoader->loadForTheme(QLatin1String("saveAs")));
```

Comment:
You have mixed icons used, on the tab context menu you the icon is "Save", here "SaveAs". I think "Save" fits better.